lib/models/project.js

```diff
@@ -66,7 +66,12 @@ Project.prototype.name = function() {
 };
 
 Project.prototype.isEmberCLIProject = function() {
-  return this.pkg.devDependencies && 'ember-cli' in this.pkg.devDependencies;
+  if (this.pkg.devDependencies && 'ember-cli' in this.pkg.devDependencies) {
```

Comment:
I believe that this can use the `dependencies` function from below, which already merges the devDeps and deps.

```javascript
return 'ember-cli' in this.dependencies();
```